<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../recursos/iconos/ic_main.png</normaloff>../recursos/iconos/ic_main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>567</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="planificarTarea">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string> Planificar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/iconos/ic_planificar.png</normaloff>../recursos/iconos/ic_planificar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarioEventos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>781</width>
       <height>461</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Cambiar el color de fondo del widget */
QCalendarWidget {
    background-color: lightblue;
}

/* Estilizar el encabezado del mes/año */
QCalendarWidget QWidget#qt_calendar_navigationbar {
    background-color: rgb(0, 85, 255);
    color: white;
}

/* Estilizar los botones de navegación */
QCalendarWidget QToolButton {
    background-color: rgb(30, 30, 30);
    color: white;
    border: 3px solid rgb(255, 255, 0);
	border-radius: 10px;
    width:100px;
    height:40px;
	margin:5px;
	icon-size: 40px 40px;
	
	font: 700 14pt &quot;Segoe UI&quot;;
}

/* Estilizar el día actual */
QCalendarWidget QDateText[current=&quot;true&quot;] {
    background-color: yellow;
    font-weight: bold;
}

/* Estilizar los fines de semana */
QCalendarWidget QDateText[weekend=&quot;true&quot;] {
    color: red;
}

/* Estilizar la celda seleccionada */
QCalendarWidget QAbstractItemView:focus {
    outline: none; /* Eliminar el borde de foco predeterminado */
}

QCalendarWidget QAbstractItemView::item:selected {
    background-color: blue;
    color: white;
}

/* Añadir un borde a las celdas de los días (esto es más complejo y puede no funcionar perfectamente en todos los estilos base) */
QCalendarWidget QAbstractItemView::item {
    border: 3px solid rgb(111, 111, 111);
} </string>
     </property>
     <property name="firstDayOfWeek">
      <enum>Qt::DayOfWeek::Monday</enum>
     </property>
     <property name="gridVisible">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QCalendarWidget::SelectionMode::SingleSelection</enum>
     </property>
     <property name="horizontalHeaderFormat">
      <enum>QCalendarWidget::HorizontalHeaderFormat::LongDayNames</enum>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::VerticalHeaderFormat::NoVerticalHeader</enum>
     </property>
     <property name="navigationBarVisible">
      <bool>true</bool>
     </property>
     <property name="dateEditEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="verEventos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string> Eventos planficados</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/iconos/eventos.png</normaloff>../recursos/iconos/eventos.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="priorizarCategorias">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: orange;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string> Gestionar categorías</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/iconos/ic_ordenarCategorias.png</normaloff>../recursos/iconos/ic_ordenarCategorias.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(40, 40, 40);
border-bottom:2px solid blue;</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuSitios">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>147</y>
      <width>153</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Turismo</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionSitios"/>
    <addaction name="actionActualidad"/>
   </widget>
   <widget class="QMenu" name="menuNotas">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Notas</string>
    </property>
    <addaction name="actionCrear"/>
    <addaction name="actionAbrir"/>
   </widget>
   <widget class="QMenu" name="menuM_s_informaci_n">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Más información</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCreditos"/>
    <addaction name="actionAyuda"/>
   </widget>
   <addaction name="menuSitios"/>
   <addaction name="menuNotas"/>
   <addaction name="menuM_s_informaci_n"/>
  </widget>
  <action name="actionSitios">
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/ic_Sitios.png</normaloff>../recursos/iconos/ic_Sitios.png</iconset>
   </property>
   <property name="text">
    <string>Sitios</string>
   </property>
   <property name="toolTip">
    <string>Ver sitios</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCrear">
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/crearNota.png</normaloff>../recursos/iconos/crearNota.png</iconset>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/abrir-documento.png</normaloff>../recursos/iconos/abrir-documento.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionCreditos">
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/informacion.png</normaloff>../recursos/iconos/informacion.png</iconset>
   </property>
   <property name="text">
    <string>Créditos</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/ayuda.png</normaloff>../recursos/iconos/ayuda.png</iconset>
   </property>
   <property name="text">
    <string>Ayuda</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionActualidad">
   <property name="icon">
    <iconset>
     <normaloff>../recursos/iconos/UbiClima.png</normaloff>../recursos/iconos/UbiClima.png</iconset>
   </property>
   <property name="text">
    <string>Actualidad</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
